DataFormats/TrackSoA/src/classes_def.xml

```diff
@@ -5,9 +5,7 @@
   <class name="reco::TrackSoAView<pixelTopology::Phase1>"/>
 
   <class name="PortableHostCollection<reco::TrackLayout<pixelTopology::Phase1>>"/>
-  <class name="pixelTrack::TracksHostPhase1" ClassVersion="3">
-    <version ClassVersion="3" checksum="794224446"/>
-  </class>
+  <class name="pixelTrack::TracksHostPhase1"/>
```

Comment:
out of curiosity (even if this lies beyond the HLT proper review) why these changes to the class version are needed?

Reply:
I followed this comment https://github.com/cms-sw/cmssw/pull/43964#discussion_r1491530401 from the previous PR. Maybe @makortel can provide more specific information about why this is the case now

Reply:
Quoting @makortel:
> Given that `TrackingRecHitHostPhase1` itself is an instantiation of a class template, its version should not be tracked this way (but with `CMS_CLASS_VERSION()` in the class template definition). But given that schema evolution of `PortableHostCollection` is unexplored anyway (and therefore nothing is promised), I'd suggest to remove the class version for now (until the schema evolution is properly understood)